psp/captureauth_test.go

```diff
@@ -0,0 +1,51 @@
+package psp
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+)
+
+func TestCapture(t *testing.T) {
+	h := testCaptureHandler{c: testHandlerCredentials{"abc", "123"}}
+
+	con := NewTestConnection(h)
+	req := CaptureAuthRequest{Amount: NewAmount(123, "USD")}
```

Comment:
can you set the AuthorizeID in the test just to demonstrate that it is required

Reply:
done